### ISSUE TITLE
New `@orama` plugins cannot be installed via npm

### ISSUE THREAD

**Issue by skoropadas** [oramasearch/orama#318]
**Describe the bug**
New `@orama` plugins cannot be installed via npm

**To Reproduce**

```
npm i @orama/plugin-parsedoc
npm i @orama/plugin-match-highlight 
```

**Expected behavior**
Should be installed correctly :)

**Screenshots**
<img width="699" alt="image" src="https://user-images.githubusercontent.com/20700969/226925890-8e028958-9491-4709-afcd-4b52702c6164.png">

**Desktop (please complete the following information):**
 - OS: MacOS

**Additional context**
Nope

**Comment by micheleriva** [comment]
Hi @skoropadas, thank you so much for noticing this. Looks like we have misconfigured our pnpm pipeline. Fixing right now!

**Comment by micheleriva** [comment]
Fixed in v1.0.0-beta.3
